docs/RELEASE-NOTES.rst

```diff
@@ -1,22 +1,38 @@
 Release Notes for BIG-IP Controller for Kubernetes
 ==================================================
 
-|release|
----------
+v1.4.0
+------
 
 Added Functionality
 ```````````````````
-* Added route domain support for the VxLAN forwarding data base addresses (FDB).
-  Also, Removed the restriction on setting the default route domain
+* Create VxLAN forwarding database (FDB) addresses for route domains.
+  Also, removed the restriction on setting the default route domain.
```

Comment:
Do users know what this restriction is? Needs clarification.

Reply:
That would be a question for @russokj

Reply:
The restriction was from the previous release note:
```
* You cannot change the default route domain for a partition managed by an F5 controller after the controller has deployed. To specify a new default route domain, use a different partition.
```

Reply:
I would break the bullet into two and remove the 'Also,' portion.  So maybe  like this:
```
* Create VxLAN forwarding database (FDB) addresses for route domains.
* Removed the restriction on setting the default route domain.
```

Reply:
I'd change this to:
```
* Enhanced route domain handling:
  - Create VxLAN forwarding database (FDB) addresses for route domains.
  - Ability to change the default route domain for a partition managed by an F5 controller after the controller has deployed.
```

Reply:
@jputrino Do you want me to add that to my commit, or are you adding it to yours?